src/components/pages/devices/flyouts/deviceDetails/propertyGrid/propertyGrid.scss

```diff
@@ -42,7 +42,7 @@ $scrollbarWidth: 4px;
   .row {
     display: flex;
     flex-direction: row;
-    @include rem-fallback(padding, 16px 0px 16px 8px);
+    @include rem-fallback(padding, 16px, 0px, 16px, 8px);
```

Comment:
Is this just a style thing? Because they work without commas and we've been doing them that way all along.  

Also, these changes are not actually related to the stated purpose of the PR... so should be held for another PR focused on cleanup type actions, if they even should be done at all.

Reply:
Nope. The current code is actually in error. It appears to work because the the fallback part of the rem-fallback mixin is preventing the error. 

`@include rem-fallback(margin, 10px 0px);`

will generate 

```
margin: 10px 0px;
margin: 10px 0px/16pxrem; <- the browser won't know what to do with this so it will default to the previous line, thus making it look like the code worked
```

while using the a comma separated parameter list will compile to the correc

```
margin: 10px 0px;
margin: 0.625rem 0rem;
```